xo-entry.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 set -e
 
-exec su-exec ${USER} "$@"
+rpcbind -f &
```

Comment:
I think `rpcbind` is added by the `openrc` package, is that explain the openrc ? What seems odd to me is that `rpcbind` is a server, using `nfsmount` require `rpcbind` ?